02-peer-protocol.md

```diff
@@ -130,6 +130,15 @@ the funding transaction and both versions of the commitment transaction.
     1. type: 0 (`upfront_shutdown_script`)
     2. data:
         * [`...*byte`:`shutdown_scriptpubkey`]
+    1. type: 1 (`channel_types`)
+    2. data:
+        * [`...*channel_type`:`types`]
+
+1. subtype: `channel_type`
+2. data:
+    * [`u16`:`len`]
```

Comment:
Because if its a u16 you can serialize an existing Features object (or at least we can), but if its a bigsize we need a separate serializer for features in this context.